docs/source/getting-started/quickstart.rst

```diff
@@ -114,7 +113,7 @@ Run the application using prototyping tools
 
 In this section, you will run the quickstart application and get introduced to the main tools for prototyping DAML:
 
-#. To compile the DAML model, run ``da run damlc -- package daml/Main.daml target/daml/iou``
+#. To compile the DAML model, run ``daml package -o target/daml/iou.dar``
```

Comment:
Can you change this to `daml build -o target/daml/iou.dar`? The package command is going away / getting hidden in favor of `build`.

Reply:
Done